model/adapter.py

```diff
@@ -723,6 +725,32 @@ def query_fun(rxn):
             reaction.bounds = 0, upper_bound
         return reaction
 
+    def minimize_distance(self):
+        """Replaces fluxomics measurements with the minimized distance"""
+        reaction_measurements = [m for m in self.measurements if m['type'] == 'reaction']
+        observations = {m['id']: np.mean(m['measurements']) for m in reaction_measurements}
+
+        uncertainties = {
+            m['id']: np.std(m['measurements']) if len(m['measurements']) >= 3 else 1
+            for m in reaction_measurements}
+
+        try:
+            # Include growth rate measurement
+            growth_rate = next(m for m in self.measurements if m['type'] == 'growth-rate')
+            growth_rate_id = constants.MODEL_GROWTH_RATE[self.model.id]
+            observations[growth_rate_id] = np.mean(growth_rate['measurements'])
+            uncertainties[growth_rate_id] = (np.std(growth_rate['measurements'])
+                                             if len(growth_rate['measurements']) >= 3 else 1)
+        except StopIteration:
+            pass
+
+        solution = adjust_fluxes2model(self.model, pd.Series(observations), pd.Series(uncertainties))
+        for reaction, minimized_distance in solution.fluxes.to_dict().items():
```

Comment:
`pandas.Series` has a method `iteritems` so that you don't have to convert `to_dict` first.